test/dummy/app/models/custom_primary_key_record.rb

```diff
@@ -0,0 +1,11 @@
+require "securerandom"
+class CustomPrimaryKeyRecord < ActiveRecord::Base
+  self.primary_key = :uuid
+
+  has_paper_trail class_name: "CustomPrimaryKeyRecordVersion"
+  default_scope -> { where(name: "custom_primary_key_record") }
```

Comment:
ok that makes sense. Please add your explanation as a comment above the `default_scope`. 

Also, please add a line in `CHANGELOG.md` and I think we'll be done here.